README.md

```diff
@@ -11,65 +11,85 @@
+```
+
+### Manual Setup
+
+#### Dependencies
+ - Ethereum Node
+   - Use the patched version of [Go Ethereum](https://github.com/makerdao/go-ethereum) (1.8.23+) in order to have access to storage transformers and diffs.
+   - [Parity 1.8.11+](https://github.com/paritytech/parity/releases)
  - [VulcanizeDB](https://github.com/makerdao/vulcanizedb)
  - Go 1.12+
  - Postgres 11.2
- - Ethereum Node
-   - [Go Ethereum](https://ethereum.github.io/go-ethereum/downloads/) (1.8.23+)
-   - [Parity 1.8.11+](https://github.com/paritytech/parity/releases)
-   
+    
 ### Getting the project
-Download the transformer codebase to your local local `GOPATH` via:
-`go get github.com/makerdao/vdb-mcd-transformers`
+Download the transformer codebase to your local local `GOPATH` via: `go get github.com/makerdao/vdb-mcd-transformers`
 
 ## Usage
-As mentioned above this repository is a plugin for VulcanizeDB.  As such it cannot be run as a standalone executable,
-but instead is intended to be included as part of a VulcanizeDB core command. There are two VulcanizeDB core commands that 
-are required for events and storage slots to be transformed and persisted to the Postgres database.
+As mentioned above this repository is a plugin for VulcanizeDB.  As such it cannot be run as a standalone executable, but instead is intended to be included as part of a VulcanizeDB core command. There are two VulcanizeDB core commands that are required for events and storage slots to be transformed and persisted to the Postgres database.
 
-1. `headerSync` fetches raw Ethereum data and syncs it into VulcanizeDB's Postgres database where then it can be used for
-transformations. More information about the `headerSync` command can be found in the [VulcanizeDB repository](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/data-syncing.md#headersync).
+1. `headerSync` fetches raw Ethereum data and syncs it into VulcanizeDB's Postgres database where then it can be used for transformations. More information about the `headerSync` command can be found in the [VulcanizeDB repository](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/data-syncing.md#headersync).
 
-1. `execute` uses the raw Ethereum data that has been synced into Postgres and applies transformations to configured MCD
-contract data via [event](./transformers/events) and [storage](./transformers/storage) transformers. The VulcanizeDB
-repository includes a [general description](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/custom-transformers.md)
-about transformers.
+1. `execute` uses the raw Ethereum data that has been synced into Postgres and applies transformations to configured MCD contract data via [event](./transformers/events) and [storage](./transformers/storage) transformers. The VulcanizeDB repository includes a [general description](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/custom-transformers.md) about transformers.
```

Comment:
Maybe doesn't need to be part of this PR, but you now also need to run `extractDiffs` to get the storage diffs used by the storage transformers

Reply:
That makes a lot of sense - probably a separate PR?